apps.json

```
@@ -4150,6 +4150,12 @@
         "state": "notworking",
         "url": "https://github.com/YunoHost-Apps/staticwebapp_ynh"
     },
+    "statping_ng": {
+        "branch": "master",
+        "revision": "c43ae9a4f32e2cbe3df47210ba9c22558071b11e",
```

Cheers !

```suggestion
        "revision": "c43ae9a4f32e2cbe3df47210ba9c22558071b11e",
```

This works but note that we encourage to not set any revision here ... otherwise you'll have to come and update this hash here each time you update the master branch

---

Ok. It could be interesting to update the add_or_update.py script because using it give me also revision